src/System.Management.Automation/engine/CommandCompletion/CommandCompletion.cs

```diff
@@ -183,6 +183,7 @@ public static CommandCompletion CompleteInput(string input, int cursorIndex, Has
                     if (remoteRunspace.GetCapabilities().Equals(Runspaces.RunspaceCapability.Default))
                     {
                         // Capability:
+                        //      CustomTransport    (0x100)          -> If remoteMachine is PowerShell 7.x or later
```

Comment:
Rather than keeping this updated as it's missing other capabilities, should we just remove this comment since the enum already describes it?

Reply:
Not sure what you mean.  We need something other than 'default' here because it falls back to v2 command completion otherwise.  I was just following current pattern.

Reply:
I mean specifically the comment block